### PR TITLE
Battle Stat on Attack Page

### PR DESCRIPTION
**Pull request by Sashank999**

### CODE REVIEW

File: torntools/scripts/content/loader/ttAttackLoader.js [hunk 1]
```diff
@@ -43,3 +43,64 @@ function warnAttackTimeout() {
...
+	console.log("Opponent Battle Stats", result.battleStatsEstimate);
+	doc.findAll("div[aria-describedby*='player-name_']").forEach((boxTitle) => {
+		const textEntries = boxTitle.find("div[class*='textEntries__']");
+		const statEstEntry = textEntries.firstElementChild.cloneNode(true);
+		statEstEntry.firstElementChild.remove();
+		statEstEntry.classList.add("tt-stat");
+		if (!mobile) {
+			if (boxTitle.getAttribute("aria-describedby").includes(`player-name_${userdata.name}`)) statEstEntry.firstElementChild.innerText = `Battle Stats: ${numberWithCommas(userdata.total, true)}`;
+			else statEstEntry.firstElementChild.innerText = `Stats Estimate: ${result.battleStatsEstimate ? numberWithCommas(result.battleStatsEstimate, true) : "Error"}`;
+		} else if (mobile) {
```
Comment (DeKleineKobini) [inline torntools/scripts/content/loader/ttAttackLoader.js:97]: Just do else here

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
It should have an option as part of stats estimate.

File: torntools/scripts/content/loader/ttAttackLoader.js (hunk 1 above)
Comment (Sashank999) [reply torntools/scripts/content/loader/ttAttackLoader.js]: Done.

**Comment by Sashank999** [comment]
Option location changed.

File: torntools/scripts/js/globalFunctions.js [hunk 2]
```diff
@@ -1615,6 +1615,7 @@ const STORAGE = {
 			stats_estimate: {
 				global: true,
 				profile: true,
+				attack_page: true,
```
Comment (DeKleineKobini) [inline torntools/scripts/js/globalFunctions.js:1618]: Not sure it should be on by default. Let's ask @Mephiles.

Comment (Sashank999) [reply torntools/scripts/js/globalFunctions.js]: Changed to off by default.

**Review by DeKleineKobini — APPROVED** [review]

**Pull request merged**